docusaurus/docs/reference/tunnelers/linux/index.md

```diff
@@ -7,9 +7,9 @@ import TabItem from '@theme/TabItem';
 
 ## The Ziti Tunneller
```

Comment:
Will you remove this heading?
a.) it looks goofy rendered imo
b.) it's not "the ziti tunneller" 

easier to just remove it imo

Reply:
Should we consolidate redundant headings like this?

## Linux Tunneller